src/generator/AutoRest.CSharp.Azure/Templates/AzureProjectJsonTemplate.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@using System.Linq
+@inherits AutoRest.Core.Template<AutoRest.CSharp.TemplateModels.ProjectJsonModel>
+{
+  "version": "@Model.Version",
+  "description": "@Model.Description",
+  "authors": [ "Microsoft" ],
+
+  "packOptions": {
+    "summary": "@Model.Description",
+    "iconUrl": "http://go.microsoft.com/fwlink/?LinkID=288890",
+    "tags": [],
+    "projectUrl": "https://github.com/Azure/azure-sdk-for-net",
+    "licenseUrl": "https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE",
+    "requireLicenseAcceptance": true
+  },
+
+  "buildOptions": {
+    "delaySign": true,
+    "publicSign": false,
+    "keyFile": "../../../../tools/MSSharedLibKey.snk"
+  },
+
+  "dependencies": {
+  },
+
+  "frameworks": {
+    "net45": {
+      "dependencies": {
+        "Microsoft.Rest.ClientRuntime.Azure": "[@Model.ClientRuntimeVersion,4.0.0)" 
+      }
+    },
+    "netstandard1.5": {
+      "imports": ["dnxcore50"],
+      "buildOptions": { "define": [ "PORTABLE" ] },
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1",
+        "System.Diagnostics.Tools": "4.0.1",
+        "System.Net.Http": "4.1.0",
+        "System.Runtime.Serialization.Primitives": "4.1.1",
+        "System.Threading.Tasks": "4.0.11",
+        "Microsoft.Rest.ClientRuntime.Azure": "[@Model.ClientRuntimeVersion,4.0.0)"
```

Comment:
What? didn't want to bother calculating the next major version!? awwwwwwww.. . 😼